lib/daru/io/exporters/excel.rb

```diff
@@ -0,0 +1,47 @@
+require 'daru/io/exporters/linkages/excel'
+
+module Daru
+  module IO
+    module Exporters
+      class ExcelHelper
+        # @note
+        #
+        # The +format+ variable used in this method, has to be given
+        # as options by the user via the +opts+ hash input.
+        #
+        # Signed off by @athityakumar on 03/06/2017 at 7:00PM
+        def prepare
+          @book  = Spreadsheet::Workbook.new
+          sheet  = @book.create_worksheet
+
+          format = Spreadsheet::Format.new color: :blue, weight: :bold
+
+          sheet.row(0).concat(@dataframe.vectors.to_a.map(&:to_s)) # Unfreeze strings
+          sheet.row(0).default_format = format
+          i = 1
+          @dataframe.each_row do |row|
```

Comment:
Wouldn't something like `each_with_index` work here? This `i += 1` looks not very idiomatic